blazor/multiselect-dropdown/templates.md

```diff
@@ -55,7 +55,7 @@ The content of each list item within the MultiSelect can be customized with the
 
 ## Value template
 
-The currently selected value that is displayed by default on the MultiSelect input element can be customized using the [ValueTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfMultiSelect-1.html#Syncfusion_Blazor_DropDowns_SfMultiSelect_1_ValueTemplate) property.
+The currently selected value that is displayed by default on the MultiSelect input element can be customized using the [ValueTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfMultiSelect-2.html#Syncfusion_Blazor_DropDowns_SfMultiSelect_2_ValueTemplate) property.
```

Comment:
redirection links are wrong. Can you cross verify all links in below places also.

Reply:
Resolved